app/src/main/java/de/westnordost/streetcomplete/quests/tactile_paving/AddTactilePavingCrosswalk.kt

```diff
@@ -22,6 +22,7 @@ class AddTactilePavingCrosswalk : OsmElementQuestType<TactilePavingCrosswalkAnsw
             !tactile_paving
             or tactile_paving = unknown
             or tactile_paving = no and tactile_paving older today -4 years
+            or tactile_paving = incorrect and tactile_paving older today -4 years
```

Comment:
```suggestion
            or tactile_paving ~ no|incorrect  and tactile_paving older today -4 years
```

Reply:
I had left it separate in case there was a discussion on how often it should be resurveyed.

Reply:
I think it belongs into the "no" category.

Adding tactile paving to a crossing that has no or incomplete tactile paving does not require the whole inersection being rebuilt.
That tactile paving gets lost is only really possible after a whole intersection is either redesigned (I'd say) or degenerates over time.